<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Question_1</class>
 <widget class="QDialog" name="Question_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>609</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>609</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image/resource_rc.qrc">
    <normaloff>:/recurce/icon.png</normaloff>:/recurce/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="text_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>248</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>609</width>
         <height>363</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Сложность &amp;quot;Падаван&amp;quot; предназначена&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;для начала обучения Python.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Выбирайте эту сложность, если у вас&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;нет знаний в Python и вы хотите &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;изучить его с нуля.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QPushButton" name="OK_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(95, 136, 240);
border-radius: 14px;</string>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image/resource_rc.qrc"/>
 </resources>
 <connections/>
</ui>
